### PR TITLE
Better introduce logs and explain them

### LINKED ISSUE #1061: Several important are missing from the Logs page
- explanation of cold and hot logs
- policy regarding hot logs becoming cold logs
- logs storage policy

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #1061

### CODE REVIEW

File: _posts/platform/app/2000-01-01-logs.md [hunk 1]
```diff
@@ -1,38 +1,45 @@
...
-```
+```shell
 scalingo --app my-app logs --lines <number of log lines>
 ```
 
-Just replace `<number of log lines>` with the amount of lines you want to display. If the logs you
-are interested in are too old, you might not be able to fetch them using the CLI.
+Replace `<number of log lines>` with the amount of lines you want to display.
+
+These live logs are called **hot logs**: they are the most recent logs of your application. When your application logged more than 50 MiB logs, Scalingo archive them in a log archives. We call them **cold logs**: they are not as recent as the hot logs but can be looked up for later analysis.
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-logs.md:26]:
```suggestion
These live logs are called **hot logs**: they are the most recent logs of your application. When your application logged more than 50 MiB logs, Scalingo archive them in a log archive. We call them **cold logs**: they are not as recent as the hot logs but can be looked up for later analysis.
```

File: _posts/platform/app/2000-01-01-logs.md [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
 nav: Logs
-modified_at: 2018-05-18 00:00:00
+modified_at: 2020-09-18 00:00:00
 tags: app logs
+index: 18
 ---
 
-Application's logs are accessible through [Scalingo CLI]({% post_url
-platform/cli/2000-01-01-features %}) or through the web dashboard in the "Logs" tab:
+Application logs can help you understand what is happening inside your application when deployed in production. Logs are especially useful for debugging problem and monitoring your application activity.
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-logs.md:9]:
```suggestion
Application logs can help you understand what is happening inside your application when deployed in production. Logs are especially useful for debugging problems and monitoring your application activity.
```

File: _posts/platform/app/2000-01-01-logs.md [hunk 3]
```diff
@@ -1,38 +1,45 @@
...
+Replace `<number of log lines>` with the amount of lines you want to display.
+
+These live logs are called **hot logs**: they are the most recent logs of your application. When your application logged more than 50 MiB logs, Scalingo archive them in a log archives. We call them **cold logs**: they are not as recent as the hot logs but can be looked up for later analysis.
 
 ## Logs Archives
 
-All your logs are archived by Scalingo for one year. When your log file grows bigger than 50M, we
-archive it. It is then available in the Logs Archive. Archived logs are not available anymore with
-the `logs` command of the CLI.
+All your logs are archived by Scalingo for one year. When your log file grows bigger than 50 MiB, Scalingo archive it. It is then available in the Logs Archive. Archived logs are no longer available with the `logs` command of the CLI.
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-logs.md:30]:
```suggestion
All your logs are archived by Scalingo for one year. When your log file grows bigger than 50 MiB, it is archived automatically. It is then available in the Logs Archive. Archived logs are no longer available with the `logs` command of the CLI.
```

File: _posts/platform/app/2000-01-01-logs.md (hunk 1 above)
Comment (yannski) [inline _posts/platform/app/2000-01-01-logs.md:26]:
```suggestion
These live logs are called **hot logs**: they are the most recent logs of your application. When your application logged more than 50 MiB logs, Scalingo archive them in a log archives. We call them **cold logs**: they are not as recent as the hot logs but can be downloaded for later analysis.
```

File: _posts/platform/app/2000-01-01-logs.md (hunk 3 above)
Comment (yannski) [inline _posts/platform/app/2000-01-01-logs.md:30]:
```suggestion
All your logs are **archived for 1 year**. When your log file grows bigger than 50 MiB, Scalingo archive it. It is then available in the Logs Archive. Archived logs are no longer available with the `logs` command of the CLI.
```

Comment (Soulou) [reply _posts/platform/app/2000-01-01-logs.md:30]: It's not true for App logs, it's about router logs for these

Comment (yannski) [reply _posts/platform/app/2000-01-01-logs.md]: @Soulou can you explain what is the log storage policy please?

Comment (Soulou) [reply _posts/platform/app/2000-01-01-logs.md]: Internal Connection Logs
→ Archived Internally for 1 year (for legal reason)

App Logs (which may include Connection logs if user checks the box)
→ Hot Logs:  up to 10 minutes (internally stored in Redis)
→ Cold Logs: up to 50Mib (internally stored on Disk)
→ Archives: Stored for 1 year officially (but we're not doing any garbage collection for now, no legal pressure here, it's our choice)

**Comment by yannski** [comment]
@EtienneM if I may, this page should be titled "Logs" and it should explain what are application logs and introduce the concept of Router logs.

File: _posts/platform/app/2000-01-01-logs.md (hunk 3 above)
Comment (yannski) [reply _posts/platform/app/2000-01-01-logs.md]: What are "Internal Connection Logs"? I've never seen this term

Comment (yannski) [reply _posts/platform/app/2000-01-01-logs.md]: What is the policy regarding addon logs?

Comment (Soulou) [reply _posts/platform/app/2000-01-01-logs.md]: Internal Connection Logs == Router Logs

It's just that we store them separately, it doesn't depend on the fact a user has enabled Router Logs on their apps or not

Comment (Soulou) [reply _posts/platform/app/2000-01-01-logs.md]: > What is the policy regarding addon logs?

Exactly the same as App (for our log service, an addon is like an app)

Comment (yannski) [reply _posts/platform/app/2000-01-01-logs.md]: Furthermore, what is the implication of activating router logs? Are the logs stored twice, once in their own storage, once mixed into the application logs?

Comment (Soulou) [reply _posts/platform/app/2000-01-01-logs.md]: > Furthermore, what is the implication of activating router logs? Are the logs stored twice, once in their own storage, once mixed into the application logs?

Yes logs are stored twice

**Pull request merged**